### PR TITLE
pAI Buffs

### PR DESCRIPTION
**Pull request by ParadoxSpace**
As in the changelog, pAI mice are now pAI rats, they now have cat and rat emotes available to them, and the pAI monkey can now rest.

### CODE REVIEW

**Comment by ParadoxSpace** [comment]
!review

File: code/modules/mob/living/silicon/pai/emote.dm [hunk 1]
```diff
@@ -86,7 +111,7 @@
 				to_chat(src, "You are not security.")
 
 		if ("help")
-			to_chat(src, "salute, bow, clap, flap, aflap, twitch, twitch_s, nod, beep, ping, \nbuzz, law, halt")
+			to_chat(src, "salute, bow, clap, flap, aflap, twitch, twitch_s, nod, beep, ping, \nbuzz, law, halt, meow1, meow2, squeal, squeak, softsqueak")
```
Comment (SoundScopes) [inline code/modules/mob/living/silicon/pai/emote.dm]: In help it's listed as "meow1", in the above code it's listed as "meow"

**Comment by Kaedwuff** [comment]
Well you're here maybe you could also fix some of the bugged modules? Like the crew manifest which won't load.

File: code/modules/mob/living/silicon/pai/emote.dm (hunk 1 above)
Comment (ParadoxSpace) [reply code/modules/mob/living/silicon/pai/emote.dm]: thank u soundscopes

**Comment by ParadoxSpace** [comment]
I'm unsure if that's within my set of skills.

**Pull request merged**